inputFiles/poromechanics/ThermoElastic_2d_base.xml

```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+    <SinglePhasePoromechanics
+      name="thermoElasticSolver"
+      solidSolverName="solidMechSolver"
+      flowSolverName="flowSolver"
+      logLevel="1"
+      isThermal="1"
+      targetRegions="{ casing, cement, rock }">
+      <NonlinearSolverParameters
+        newtonMaxIter="100"/>
+      <LinearSolverParameters
+        directParallel="0"
+        logLevel="1"/>
+    </SinglePhasePoromechanics>
+
+    <SolidMechanicsLagrangianSSLE
+      name="solidMechSolver"
+      timeIntegrationOption="QuasiStatic"
+      logLevel="1"
+      discretization="FE1"
+      targetRegions="{ casing, cement, rock }"/>
+   
+    <SinglePhaseFVM
+      name="flowSolver"
+      logLevel="1"
+      discretization="tpfaFlow"
+      temperature="0"
+      isThermal="1"

```

Comment:
let's have the `SinglePhasePoromechanics` overrule this so that the user only needs to set the flag in the `coupledSolver`.

Reply:
Done, setting this `isThermal` flag is not necessary anymore in the flow solver